### PR TITLE
Try to speed up documentation build

### PR DESCRIPTION
**Pull request by glwagner**
First thing is simply to use more threads.

But if we can uses parallelism at a little higher level we might do better.

### CODE REVIEW

**Comment by glwagner** [comment]
The tests have sped up a lot (presumably because compile time is so much better with KA 0.9). This means that the docs build is a much bigger bottleneck. I think we're at 2h48m:

https://buildkite.com/clima/oceananigans/builds/11123#01879a9c-fde8-4691-a3f5-29ef6060e288

so we will see what we get down to.

**Comment by navidcy** [comment]
It had some speedup effect…

**Review by navidcy — COMMENTED** [review]
mutlithreading the docs built has no effect...

**Comment by glwagner** [comment]
Yea weird.

**Comment by glwagner** [comment]
Trying this now:

https://github.com/glwagner/Documenter.jl/blob/225b7cdadd451ed43ecdf5078439d72fad5b9994/src/Expanders.jl#L37

Though, it may be better to use `asyncmap` or something. But we'll see what happens this time around.

File: docs/Project.toml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 [deps]
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-DocumenterCitations = "daee34ce-89f3-4625-b898-19384cb65244"
```
Comment (navidcy) [inline docs/Project.toml:4]: probably removing DocumenterCitations will make docs built to fail

Comment (navidcy) [reply docs/Project.toml:4]: I see the problem probably is that you "upgraded" Documenter to v0.28 in your fork and then DocumenterCitations can't handle it due to compat restriction here --> https://github.com/ali-ramadhan/DocumenterCitations.jl/blob/5ef4fbf4b71cc04941ad5258adea666c046b73da/Project.toml

So perhaps just add back the `-DEV` in the Documenter version for these experiments?

Comment (glwagner) [reply docs/Project.toml:4]: Should I have changed the version on my fork to 0.27? I'm confused what the right process is here

Comment (navidcy) [reply docs/Project.toml:4]: That would also worked!! :)

Comment (navidcy) [reply docs/Project.toml:4]: Do that. Then no changes in DocumenterCitation are needed

**Comment by navidcy** [comment]
@glwagner did you try building the docs locally? If they don’t build locally then it’s futile.

**Comment by navidcy** [comment]
Shall we close this PR in favor of #3135?

**Comment by glwagner** [comment]
ah yea for sure

**Pull request closed without merging**